subprojects/delivery/nupokati/src/main/kotlin/com/avito/android/model/CdBuildConfig.kt

```diff
@@ -0,0 +1,45 @@
+package com.avito.android.model
+
+public data class CdBuildConfig(
+    val schemaVersion: Long,
+    val project: NupokatiProject,
+    val outputDescriptor: OutputDescriptor,
+    val releaseVersion: String,
+    val deployments: List<Deployment>
+) {
+
+    public data class OutputDescriptor(
+        val path: String,
+        val skipUpload: Boolean
+    )
+
+    public sealed class Deployment {
+
+        public data class GooglePlay(
+            val artifactType: AndroidArtifactType,
+            val buildVariant: BuildVariant,
+            val track: Track
+        ) : Deployment()
+
+        /**
+         * Deploy artifacts to QApps.
+         * Uses UploadToQapps build step to find them.
+         */
+        public data class Qapps(
+            /**
+             * Send artifacts as release versions.
+             * Non-release artifacts stored limited time.
```

Comment:
```suggestion
             * Non-release artifacts are stored limited time.
```

Reply:
also `for a limited time` is better, right?

Reply:
yes